app/routes/docs.notices.schema.mdx

```diff
@@ -12,47 +12,63 @@ export function loader() {
   return feature('SCHEMA') || new Response(null, { status: 404 })
 }
 
-# GCN Unified Schema
+# Unified Schema
```

Comment:
Unified looks general, what about "Unified Schema for GCN"?

Reply:
It's on the GCN web site, isn't that implied?

Reply:
Ya that also I felt, but second thought that was self-consistent.